easybuild/easyconfigs/t/TN93/TN93-1.0.7-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,50 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Iñaki Mtz de Ilarduya Muñoz
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = "CMakeMake"
+
+name = 'TN93'
+version = '1.0.7'
+
+homepage = 'https://github.com/veg/tn93'
+description = """This is a simple program meant to compute pairwise distances between
+ aligned nucleotide sequences in sequential FASTA format using the Tamura Nei 93 distance."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/veg/tn93/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4a088689a23e8dc92d4484431ab0de2a0f5f05109d2a56e7bd55c0cc6459c4db']
+
+unpack_options = '--strip-components=1'
+
+separate_build_dir = True
+
+buildininstalldir = True
```

Comment:
I don't see any reason why you would do this, as you are copying everything into the correct installation directory anyway.